typescript/infra/config/environments/testnet2/agent.ts

```diff
@@ -71,7 +71,10 @@ export const abacus: AgentConfig<TestnetChains> = {
   relayer: {
     default: {
       signedCheckpointPollingInterval: 5,
-      blacklist: releaseCandidateHelloworldMatchingList,
+      blacklist: [
+        ...releaseCandidateHelloworldMatchingList,
+        { destinationAddress: '0xBC3cFeca7Df5A45d61BC60E7898E63670e1654aE' },
```

Comment:
Just wanna confirm that omitting the other fields equates to a wildcard?

Reply:
Pretty sure